plugins/base/src/main/kotlin/signatures/JvmSingatureUtils.kt

```diff
@@ -0,0 +1,129 @@
+package org.jetbrains.dokka.base.signatures
+
+import javaslang.Tuple2
+import org.jetbrains.dokka.base.translators.documentables.PageContentBuilder
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.model.properties.WithExtraProperties
+
+interface JvmSingatureUtils {
+
+    fun PageContentBuilder.DocumentableContentBuilder.annotationsBlock(d: Documentable)
+
+    fun PageContentBuilder.DocumentableContentBuilder.annotationsInline(d: Documentable)
+
+    fun <T : Documentable> WithExtraProperties<T>.modifiers(): Set<ExtraModifiers>
+
+    fun Set<ExtraModifiers>.toSignatureString(): String =
+        joinToString("") { it.name.toLowerCase() + " " }
+
+    fun <T : Documentable> WithExtraProperties<T>.annotations(): List<Annotations.Annotation> =
+        extra[Annotations]?.content ?: emptyList()
+
+    private fun Annotations.Annotation.toSignatureString(): String =
+        "@${this.dri.classNames}(${this.params.entries.joinToString { it.key + "=" + it.value }})"
```

Comment:
You don't have to use string interpolation here as it's unreadable

Reply:
If you think so...